docs/lang/articles/misc/gui.md

```diff
@@ -271,3 +471,53 @@ draw()
 
 ti.imwrite(pixels, f"export_f32.png")
 ```
+
+[`ti.imread(filename)`](https://api-docs.taichi.graphics/src/taichi.misc.html?highlight=imread#taichi.misc.image.imread)
+loads an image from the target filename and returns it as a `np.ndarray(dtype=np.uint8)`.
+Each value in this returned field is an integer in [0, 255].
+
+[`ti.imshow(img, windname)`](https://api-docs.taichi.graphics/src/taichi.misc.html?highlight=imshow#taichi.misc.image.imshow)
+creates an instance of ti.GUI and show the input image on the screen. It has the same logic as `ti.imwrite` for different data types.
+
+[`ti.imresize(img, w)`](https://api-docs.taichi.graphics/src/taichi.misc.html?highlight=imresize#taichi.misc.image.imresize)
+resizes the img specified.
+
+## Zero-copying frame buffer
+When the GUI resolution (window size) is large, it sometimes becomes difficult to achieve 60 FPS even without any kernel
+invocations between two frames.
+
+This is mainly due to the copy overhead, where Taichi GUI needs to copy the image buffer from one place to another.
+This copying is necessary for the 2D drawing functions, such as `gui.circles`, to work. The larger the image shape is,
```

Comment:
"This copying is necessary for the 2D drawing functions, such as `gui.circles`, to work."
strange sentence.